src/main.rs

```diff
@@ -156,7 +156,14 @@ fn main() {
         std::process::exit(1);
     }
 
-    let ulimit: rlimit::rlim = adjust_ulimit_size(&opts);
+    // Rlimit does not support Windows
+    // set to 1000 if Windows is used
+    let ulimit: rlimit::rlim = if !(cfg!(windows)) {
+        adjust_ulimit_size(&opts)
+    } else {
+        1000
```

Comment:
Can we extract this to a constant? Similar to what we are doing for the ports. :)

Reply:
I can make it a constant, not sure on what you mean by extract though?